procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "3dbf3c0c",
```

Comment:
<p>Add some context here for why this happens.</p><p>The current DDS configuration results in history being lost when CSCs are restarted and/or a small network outage occurs. If this happens, then to receive the summaryState event, which is required by salobj.get_summary_state (and used extensively in the atcs), it is necessary to manually change the state of a CSC. The commands below can be used to perform this action.</p><p>A new DDS configuration is now ready to be tested and will hopefully be released in the near future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Moved these commands to a Troubleshooting Section at the end (specially since this condition will soon disappear)</p>